src/Umbraco.Web.UI.Client/src/views/common/dialogs/login.html

```diff
@@ -40,7 +40,10 @@
                 </div>
 
                 <div class="control-group" ng-class="{error: loginForm.password.$invalid}">
-                    <input type="password" ng-model="password" name="password" class="input-xlarge" localize="placeholder" placeholder="@placeholders_password" autocomplete="off" />
+                    <input type="{{showpassword ? 'text' : 'password'}}" ng-model="password" name="password" class="input-xlarge password" ng-class="{'has-password': password.length > 0}" localize="placeholder" placeholder="@placeholders_password" autocomplete="off" />

```

Comment:
We need to double check that setting 'type' dynamically with interpolation will work in all browsers. For example, interpolated attributes in IE for things like 'class' don't work. You might have to use something like ng-attr-type to be sure.